src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -666,11 +667,6 @@ private function handleOutOfStockProduct(OrderForViewing $orderForViewing)
      */
     public function addProductAction(int $orderId, Request $request): Response
     {
-        /** @var OrderForViewing $orderForViewing */
-        $orderForViewing = $this->getQueryBus()->handle(new GetOrderForViewing($orderId));
```

Comment:
I know we don't need this any more since we don't rely on the response content, but it is still a BC break
I suggest we wait until the RC1 is validated (or not) If an RC2 is required then we can remove this code, if not let's leave it with a comment explaining that it's not used any more and should be removed in a major version